lib/workers/repository/updates/generate.ts

```diff
@@ -115,7 +115,7 @@ export function generateBranchConfig(
         upgrade.newDigestShort ||
         upgrade.newDigest.replace('sha256:', '').substring(0, 7);
     }
-    if (upgrade.isDigest) {
+    if (upgrade.isPin) {
```

Comment:
@rarkins @viceice This here made me do the functional change.
It was `if(upgrade.isDigest)` before and its undefined when the user uses `pinDigests` flag.
It was setting the change in the table to the tag instead of the digests.

Reply:
@rarkins @viceice Another possibility to fix this without changing much is to set the updateType back to `digest`. And then here we change the checking to `upgrade.updateType`.

Reply:
Yes, would it work to check for `if (upgrade.updateType === 'digest' and !upgrade.currentDigest)` ?

Another possibility is introducing an updateType of `pin-digest`, which maybe should have already existed.

Reply:
Yes that would work. But lets do the `pin-digest` type ? sounds more natural in my opinion.

Reply:
I think it's more correct.. but I'm not sure how much cascading changes it would cause. For example we still want it to be part of any "Pin dependencies" PR